electron/main/Files/registerFilesHandler.ts

```diff
@@ -43,7 +43,12 @@ export const registerFileHandlers = () => {
 
   ipcMain.handle(
     "write-file",
-    async (event, filePath: string, content: string) => {
+    async (
+      event,
+      filePath: string,
+      content: string,
+      indexFileToo: boolean = false
+    ) => {
```

Comment:
Nit: if we define the props as an object, when passing arguments to the function, the usage would include the name of the prop. 

This makes it easier to read the code IMO

Reply:
good suggestion